yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1183,11 +1194,22 @@ object Client extends Logging {
   private def getUserClasspath(
```

Comment:
This method is not needed anymore, is it? Only the one that takes a `SparkConf` directly.